etcdutl/etcdutl/printer_fields.go

```diff
@@ -27,4 +27,5 @@ func (p *fieldsPrinter) DBStatus(r snapshot.Status) {
 	fmt.Println(`"Revision" :`, r.Revision)
 	fmt.Println(`"Keys" :`, r.TotalKey)
 	fmt.Println(`"Size" :`, r.TotalSize)
+	fmt.Println(`"Version" :`, r.Version)
```

Comment:
What does it prints when speaks to etcd v3.4 ?

Reply:
empty string.
```
etcd $ ./etcd-v3.4.16/etcdctl version
etcdctl version: 3.4.16
API version: 3.4
etcd $ ./etcd-v3.4.16/etcdctl snapshot save 1.db
Snapshot saved at 1.db
etcd $ ./bin/etcdutl version
etcdutl version: 3.5.0-alpha.0
API version: 3.5
etcd $ ./bin/etcdutl snapshot status 1.db 
bbfa5a14, 0, 3, 20 kB, 
etcd $ ./bin/etcdutl snapshot status 1.db -w table
+----------+----------+------------+------------+---------+
|   HASH   | REVISION | TOTAL KEYS | TOTAL SIZE | VERSION |
+----------+----------+------------+------------+---------+
| bbfa5a14 |        0 |          3 |      20 kB |         |
+----------+----------+------------+------------+---------+
etcd $ ./bin/etcdutl snapshot status 1.db -w json
{"hash":3153746452,"revision":0,"totalKey":3,"totalSize":20480,"version":""}
```